src/lib/import/import.service.ts

```diff
@@ -361,13 +361,13 @@ export class ImportService {
     }
 
     private importTransitionEvents(xmlTrans: Element, trans: Transition, result: PetriNetResult) {
-        try {
-            const events = Array.from(xmlTrans.getElementsByTagName('event'));
-            events.forEach((xmlEvent, index) => {
-                this.importTransitionEvent(xmlEvent, index, trans, result);
-            });
-        } catch (e) {
-            result.addError('Importing transition events failed', e as Error);
+        const events = Array.from(xmlTrans.getElementsByTagName('event'));
+        for (const xmlEvent of events) {
```

Comment:
`importTransitionEvent` has whole body in try-catch, there is no need to try-catch the call, old implementation was ok